tests/scripts/deploy_admission_controller.sh

```diff
@@ -54,6 +54,7 @@ echo "Deploying cert-manager"
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
 timeout 150 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
 timeout 20 sh -c 'until [ $(kubectl -n cert-manager get pods -o custom-columns=READY:status.containerStatuses[*].ready | grep -c true) -eq 3 ]; do sleep 1; done'
+sleep 20
```

Comment:
How can we check like above instead of sleeping?

Reply:
the reason for sleep is for cainjector to inject ca certs https://github.com/jetstack/cert-manager/issues/3338#issuecomment-707579834. I'm not aware if there is any command to check ca injector status.

Reply:
I'm looking if I can find something

Reply:
found a way.